geminimd/commands.py

```
@@ -0,0 +1,401 @@
+            removed = 0
+            for path in to_delete:
+                try:
+                    if path.is_dir():
+                        import shutil
+
+                        shutil.rmtree(path)
+                    else:
+                        path.unlink()
+                    removed += 1
+                except Exception:
+                    env.ui.console.print(f"[red]Failed to remove {path}")
```

Ruff (0.15.9)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)